python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -646,6 +654,7 @@ def __init__(
         writer: Optional[ray.actor.ActorHandle],
         reader_and_node_list: List[Tuple["ray.actor.ActorHandle", str]],
         num_shm_buffers: int,
+        read_by_multi_output_node: bool,
```

Comment:
add doc string

Reply:
done. and updated the var name to read_by_adag_driver